src/configure.rs

```diff
@@ -320,7 +320,14 @@ fn get_editor_internal(visual: Option<String>, editor: Option<String>) -> String
 
 fn get_config_path() -> OsString {
     if let Some(config_path) = env::var_os("STARSHIP_CONFIG") {
-        return config_path;
+        let mut paths = env::split_paths(&config_path);
```

Comment:
I noticed that there was one more place where `$STARSHIP_CONFIG` was read: in editing or toggling values when invoked as a CLI. Does this look sensible? I could bubble up the error but it'd require more invasive changes as this function is called from multiple entrypoints that do not return a Result themselves.